plugins/module_utils/_auth_method_k8s.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 FERREIRA Christophe (@chris93111)
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+'''Python versions supported: all controller-side versions, all remote-side versions except 2.6'''
```

Comment:
```suggestion
'''Python versions supported: >=3.6'''
```
This is a good reminder I need to update these in the other files.